ui/src/main/java/com/xmartlabs/base/ui/module/AndroidModule.java

```diff
@@ -0,0 +1,28 @@
+package com.xmartlabs.base.ui.module;
+
+import android.app.Application;
+import android.content.Context;
+
+import javax.inject.Singleton;
+
+import dagger.Module;
+import dagger.Provides;
+
+/**
+ * Created by mike on 10/04/2017.
+ */
+@Module
+public class AndroidModule {
```

Comment:
This actually is. That's why i suggest using a single `Injector` that handles all the injections